packages/typescript-estree/src/parseSettings/createParseSettings.ts

```diff
@@ -136,6 +162,17 @@ export function createParseSettings(
     });
   }
 
+  // No type-aware linting which means that cross-file (or even same-file) JSDoc is useless
+  // So in this specific case we default to 'none' if no value was provided
+  if (
+    options.jsDocParsingMode == null &&
+    parseSettings.projects.length === 0 &&
+    parseSettings.programs == null &&
+    parseSettings.EXPERIMENTAL_projectService
```

Comment:
Should this be `!parseSettings.EXPERIMENTAL_projectService`?

Reply:
![goodcatch](https://github.com/typescript-eslint/typescript-eslint/assets/7462525/5fc70636-b023-4710-9ac2-3246f10e2c9c)